boards/m4a-wrover/include/board.h

```diff
@@ -0,0 +1,182 @@
+/*
+ * Copyright (C) 2018 Gunar Schorcht
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     boards_m4a-wrover
+ * @brief       Board specific definitions for Espressif ESP-WROVER-KIT V3
+ * @{
+ *
+ * The Espressif ESP-WROVER-KIT is a development board that uses the
+ * ESP32-WROVER module which includes a built-in 4 MByte SPI RAM. Most
+ * important features of the board are
+ *
+ * - Micro-SD card interface
+ * - OV7670 camera interface
+ * - 3.2" SPI LCD panel
+ * - RGB LED
+ *
+ * Furthermore, many GPIOs are broken out for extension. The USB bridge
+ * based on FDI FT2232HL provides a JTAG interface for debugging through
+ * the USB interface.
+ *
+ * When the camera module is connected, add
+ * ```
+ * USEMODULE += esp32_wrover_kit_camera
+ * ```
+ * to the makefile of the application to use the according default board
+ * configuration.
+ *
+ * For detailed information about the configuration of ESP32 boards, see
+ * section ref esp32_peripherals "Common Peripherals".
```

Comment:
for example this txt section can fit in the doc.txt, i think...

Reply:
this was changed to doc.txt